### PR TITLE
tus.io feedback needed

### PR DESCRIPTION
**Pull request by bgruening**
@gmauro @gallardoalba  what do you think?

### CODE REVIEW

**Review by gmauro — APPROVED** [review]

File: galaxy/ftp.md [hunk 1]
```diff
@@ -4,9 +4,21 @@ title: FTP service instructions
...
 ---
 
-# UseGalaxy.eu FTP Server
+-------
+
+## We need your help and feedback ✨ 
+
+We integrated a new way to upload data into Galaxy based on the [tus.io](https://tus.io/). This will enable our users to upload
+large amounts of data via the web-interface in a more reliable way and with resumable file upload support. This also means that the 1GB limit
+for uploads via your browser does __not__ exist anmore ✨ . If this scales well over the next month and you are happy, we would like to retire
```
Comment (beatrizserrano) [inline galaxy/ftp.md:13]:
```suggestion
for uploads via your browser does __not__ exist anymore ✨ . If this scales well over the next month and you are happy, we would like to retire
```

File: galaxy/ftp.md [hunk 2]
```diff
@@ -4,9 +4,21 @@ title: FTP service instructions
...
+the FTP server and encourage all our users to use the normal upload tool in Galaxy.
+
+Try our the new upload and [let us know how it goes]](mailto:galaxy@informatik.uni-freiburg.de?subject=Tus.io+feedback).
```
Comment (beatrizserrano) [inline galaxy/ftp.md:16]: contact@usegalaxy.eu?

Comment (bgruening) [reply galaxy/ftp.md:16]: Mh yes, took it from the page further below. Need to change both I think.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**